### PR TITLE
Orientation flexbox fix

### PR DESCRIPTION
**Pull request by atfo**

### CODE REVIEW

File: common/src/Schedule/Course.tsx [hunk 1]
```diff
@@ -51,24 +51,25 @@ export function PortraitCourse({ course, day }: { course: ScheduleEntry, day: nu
 }
 
 export function LandscapeCourse({ course, day }: { course: ScheduleEntry, day: number }) {
-    const oneHourHeight = (ScreenService.getScreenSize().height - 64 - 8) / 10;
+    const oneHourHeight = (ScreenService.getScreenSize().height - 64 - 8) / 11;
+        // (ScreenService.getScreenSize().height - 64)/9;
```
Comment (markspolakovs) [inline common/src/Schedule/Course.tsx]: Remove useless commented lines pls

File: common/lib/msm/schedule.ts [hunk 2]
```diff
@@ -37,6 +37,7 @@ export default async (start: number, end: number): Promise<GetScheduleResponse>
         end,
     });
     const data = (await response.json()) as ScheduleEntry[];
+    // console.warn('schedule'+JSON.stringify(data));
```
Comment (markspolakovs) [inline common/lib/msm/schedule.ts]: Remove

File: common/src/Schedule/Course.tsx [hunk 3]
```diff
@@ -51,24 +51,25 @@ export function PortraitCourse({ course, day }: { course: ScheduleEntry, day: nu
...
     const oneMinuteHeight = oneHourHeight / 60;
     const oneDayWidth = (ScreenService.getScreenSize().width / 5);
     const start = moment(course.start.substring(0, course.start.length - 1));
     const period = parseInt(course.param_2.replace(/Period ([1-9]) {10}-/, '$1'), 10);
     const style = [
         GlobalStyles.schedule.landscapeCourse,
         {
-            left: ((day - 1) * oneDayWidth) + 2,
+            // left: ((day - 1) * oneDayWidth) + 2,
```
Comment (markspolakovs) [inline common/src/Schedule/Course.tsx]: Remove

File: common/src/Schedule/Day.tsx [hunk 4]
```diff
@@ -23,7 +23,7 @@ function getHours() {
     return ['08:00', '09:00', '10:00', '11:00', '12:00', '13:00', '14:00', '15:00', '16:00']
         .map((hour, index) => {
             const oneHourHeight = (ScreenService.getScreenSize().height - 64 - 8) / 10;
-            const top = 64 + (oneHourHeight * (index));
+            const top = (oneHourHeight * (index)) - (oneHourHeight)/4;
```
Comment (markspolakovs) [inline common/src/Schedule/Day.tsx]: Useless parens.

File: common/src/Schedule/component.tsx [hunk 5]
```diff
@@ -50,98 +49,79 @@ function LandscapeSchedule({ schedule }: { schedule: ScheduleEntry[] }) {
...
         },
     };
-
     constructor(props: ScheduleProps) {
         super(props);
         const dimens = Dimensions.get('window');
         this.state = {
             landscape: dimens.width > dimens.height,
         };
+        console.log(this.props.schedule);
```
Comment (markspolakovs) [inline common/src/Schedule/component.tsx]: Remove

File: common/src/Schedule/state.ts [hunk 6]
```diff
@@ -32,7 +32,7 @@ const loadSchedule = (start: number = null, end: number = null) => async (dispat
             .set({ hour: 0, minute: 0, second: 0 })
             .unix(),
         end || moment() // last day of this week
-            .isoWeekday(7)
+            .isoWeekday(5) // from 7 to 5
```
Comment (markspolakovs) [inline common/src/Schedule/state.ts]: Unnecessary comment

File: common/styles.js [hunk 7]
```diff
@@ -190,13 +190,15 @@ const normalStyles = {
             flex: 1,
             flexDirection: 'row',
             backgroundColor: colors.background,
+            // justifyContent: 'flex-start',
```
Comment (markspolakovs) [inline common/styles.js]: Remove

File: common/styles.js [hunk 8]
```diff
@@ -190,13 +190,15 @@ const normalStyles = {
...
         },
         landscapeScheduleContainer: {
             padding: 0,
             margin: 0,
             top: 0,
             left: 0,
-            display: 'flex',
+            // display: 'flex',
```
Comment (markspolakovs) [inline common/styles.js]: Remove

File: common/styles.js [hunk 9]
```diff
@@ -256,27 +259,19 @@ const normalStyles = {
         },
         landscapeCourse: {
             position: 'absolute',
-            marginRight: 16,
+            // marginRight: 16,
```
Comment (markspolakovs) [inline common/styles.js]: Remove

File: common/styles.js [hunk 10]
```diff
@@ -256,27 +259,19 @@ const normalStyles = {
...
             borderRadius: 2,
-            paddingLeft: 8,
+            // paddingLeft: 8,
             paddingTop: 0,
             elevation: 1,
-            marginBottom: 16,
+            // marginBottom: 16,
```
Comment (markspolakovs) [inline common/styles.js]: Remove

File: common/styles.js [hunk 11]
```diff
@@ -256,27 +259,19 @@ const normalStyles = {
         },
         landscapeCourse: {
             position: 'absolute',
-            marginRight: 16,
+            // marginRight: 16,
             borderRadius: 2,
-            paddingLeft: 8,
+            // paddingLeft: 8,
```
Comment (markspolakovs) [inline common/styles.js]: Remove

File: common/src/Schedule/state.ts (hunk 6 above)
Comment (atfo) [inline common/src/Schedule/state.ts]: That was for you to see on the screenshot.

**Comment by atfo** [comment]
I'll deal with all this in 40'.

**Comment by atfo** [comment]
All neat and clean

**Comment by markspolakovs** [comment]
Linter is still angry:

```
ERROR: 6:5    no-unused-variable  'Orientation' is declared but never used.
ERROR: 20:11  no-unused-variable  'min' is declared but never used.
ERROR: 21:11  no-unused-variable  'max' is declared but never used.
ERROR: 22:11  no-unused-variable  'isLandscape' is declared but never used.

common/src/index.tsx
ERROR: 22:11  no-unused-variable  'AppState' is declared but never used.
```

**Pull request merged**